test/package.json

```diff
@@ -5,8 +5,8 @@
   "main": "index.js",
   "license": "MIT",
   "dependencies": {
-    "@callstack/eslint-config": "..",
-    "eslint": "^5.4.0",
-    "react": "^16.4.2"
+    "@callstack/eslint-config": "*",
```

Comment:
🤘🏻movin too fast

Reply:
I'm linking it properly with yarn now. Using `file://` protocol results in Yarn installing projects _very_ slow (they're duplicated in cache as well). Not sure if that's desired behavior or a bug tho. Linking works in this case and it's fast enough :D